esmvaltool/cmorizers/obs/cmor_config/NSIDC-0116-nh.yml

```
@@ -0,0 +1,26 @@
+---
+# Common global attributes for Cmorizer output
+attributes:
+  dataset_id: NSIDC-0116-nh
+  version: '4.1'
+  tier: 2
```

```suggestion
  tier: 3
```
Since registration is required for downloading the data.